### DISCUSSION TITLE
Install CUDA version of llama-cpp-python package

### DISCUSSION THREAD

**Discussion by Fissium** [astral-sh/rye#469] [Q&A]
Hi. I have to use [llama-cpp-python](https://github.com/abetlen/llama-cpp-python/)  package in my project. However,  `rye add llama-cpp-python` and `rye sync` install cpu version of this package. To install CUDA version I should use the following:
`CMAKE_ARGS="-DLLAMA_CUBLAS=on" pip install llama-cpp-python`
Is it possible to do using rye?

**Comment by mikefaille** [comment]
I answered here : https://github.com/astral-sh/rye/issues/1271#issuecomment-2325408528
